content/_data/upgrades/2-221-1.adoc

```diff
@@ -0,0 +1,55 @@
+==== Header and Breadcrumb Layout Improvements
+
+Instances with plugins that depend on specific details of the Jenkins layout (like the https://plugins.jenkins.io/simple-theme-plugin/[Simple Theme plugin]) may have user interface issues with this release.
+
+==== Security Defaults
+
+Global security settings no longer have a user interface setting that will disable them.
+Previously, administrators could disable security with a single checkbox.
+They could also disable cross-site request forgery (CSRF) protection with a single checkbox.
+Those two settings **cannot be disabled** in this release.
```

Comment:
Maybe worth mentioning that some abandoned, deprecated, or proprietary plugins may still not be compatible with CSRF protection and would need to be fixed.